### PR TITLE
Rename Tutorial and changes to getting-started doc

### PR DESCRIPTION
**Pull request by gabcoyne**
Tutorials vs guides sounds redundant, so I was looking at other docs and vue.js has a single tutorial, and then a guide. We similarly have really only one tutorial, end to end, and several guides. PR renames tutorials to tutorial.

### CODE REVIEW

**Comment by discdiver** [comment]
Sweet! You'll want to add redirects to netlify.toml to avoid 404s, probably with a [splat](https://docs.netlify.com/routing/redirects/redirect-options/#splats)

**Review by discdiver — CHANGES_REQUESTED** [review]
I'm good with renaming "tutorials" to "tutorial"
- making a new worker was moved from tutorial to guide - merging from main branch should update that
- noted netlifly.toml redirects need created in another comment
- there's a disconnect from having the projects page and then the following step is the deployments page that doesn't mention projects and makes it sound like deployments haven't been introduced. Maybe that's fine while projects are in beta, assuming they move to GA before long. That's not something your changes introduced, just noting it.

**Comment by gabcoyne** [comment]
@biancaines from the above @discdiver is echoing your concern about order of projects and deployments

File: docs/cloud/cloud-quickstart.md [hunk 1]
```diff
@@ -135,19 +135,19 @@ $ python quickstart_flow.py
...
 
-![Viewing local flow run details in the Prefect Cloud UI.](../img/ui/cloud-flow-run-details.png)
+![Viewing local flow run details in the Prefect Cloud UI.](/img/ui/cloud-flow-run-details.png)
 
 Congratulations! You successfully ran a local flow and, because you're logged into Prefect Cloud, the local flow run results were captured by Prefect Cloud.
 
 ## Next steps
 
-If you're new to Prefect, learn more about writing and running flows in the [Prefect Flows First Steps](/tutorials/first-steps/) tutorial. If you're already familiar with Prefect flows and want to try creating deployments and kicking off flow runs with Prefect Cloud, check out the [Deployments](/tutorials/deployments/) and [Storage and Infrastructure](/tutorials/storage/) tutorials.
+If you're new to Prefect, learn more about writing and running flows in the [Prefect Flows First Steps](/tutorial/first-steps/) tutorial. If you're already familiar with Prefect flows and want to try creating deployments and kicking off flow runs with Prefect Cloud, check out the [Deployments](/tutorial/deployments/) and [Storage and Infrastructure](/docs/tutorial/storage/) tutorials.
```
Comment (serinamarie) [inline docs/cloud/cloud-quickstart.md:150]:
```suggestion
If you're new to Prefect, learn more about writing and running flows in the [Prefect Flows First Steps](/tutorial/first-steps/) tutorial. If you're already familiar with Prefect flows and want to try creating deployments and kicking off flow runs with Prefect Cloud, check out the [Deployments](/tutorial/deployments/) and [Storage and Infrastructure](/tutorial/storage/) tutorials.
```

File: docs/cloud/connecting.md [hunk 2]
```diff
@@ -98,7 +98,7 @@ In this example, we configured `PREFECT_API_URL` and `PREFECT_API_KEY` in the de
 !!! note "Environment variables"
     You can also set `PREFECT_API_URL` and `PREFECT_API_KEY` as you would any other environment variable. See [Overriding defaults with environment variables](/concepts/settings/#overriding-defaults-with-environment-variables) for more information.
 
-See the [Flow orchestration with Prefect](/tutorials/orchestration/) tutorial for examples.
+See the [Flow orchestration with Prefect](/orchestration/) tutorial for examples.
```
Comment (serinamarie) [inline docs/cloud/connecting.md:101]:
```suggestion
See the [Flow orchestration with Prefect](/tutorial/orchestration/) tutorial for examples.
```

File: docs/cloud/index.md [hunk 3]
```diff
@@ -15,12 +15,12 @@ title: Prefect Cloud
 
 # Welcome to Prefect Cloud <span class="badge cloud"></span>
 
-Prefect Cloud is a workflow coordination-as-a-service platform. Prefect Cloud provides all the capabilities of the [Prefect server](/tutorials/orchestration/#running-the-prefect-server) and UI in a hosted environment, plus additional features such as automations, workspaces, and organizations.
+Prefect Cloud is a workflow coordination-as-a-service platform. Prefect Cloud provides all the capabilities of the [Prefect server](/orchestration/#running-the-prefect-server) and UI in a hosted environment, plus additional features such as automations, workspaces, and organizations.
```
Comment (serinamarie) [inline docs/cloud/index.md:18]:
```suggestion
Prefect Cloud is a workflow coordination-as-a-service platform. Prefect Cloud provides all the capabilities of the [Prefect server](/orchestration/#running-the-prefect-server) and UI in a hosted environment, plus additional features such as automations, workspaces, and organizations.
```
```suggestion
Prefect Cloud is a workflow coordination-as-a-service platform. Prefect Cloud provides all the capabilities of the [Prefect server](/tutorial/orchestration/#running-the-prefect-server) and UI in a hosted environment, plus additional features such as automations, workspaces, and organizations.
```

**Review by serinamarie — CHANGES_REQUESTED** [review]

**Comment by gabcoyne** [comment]
@serinamarie All links pass

File: docs/index.md [hunk 4]
```diff
@@ -35,11 +35,11 @@ Trying to implement these features from scratch is a huge pain that takes time,
 </figure>
 
 #### New to Prefect?
-If you're ready to dive in and learn Prefect, check out the Prefect [concepts](/concepts/index/) and try some [tutorials](/tutorials/) to see how they work in action.
+If you're ready to dive in and learn Prefect, check out the Prefect [concepts](/concepts/index/) and try some [tutorials](/) to see how they work in action.
 
 For deeper dives on specific topics, explore our [guides](guides/index/) for common use-cases. <div style="height: 10px"></div>
 
-[Concepts](/concepts){ .md-button .main-button--secondary .full } [Tutorials](/tutorials/){ .md-button .md-button--primary .main-button--primary .full }  [Guides](guides){ .md-button .main-button--secondary .full }
+[Concepts](/concepts){ .md-button .main-button--secondary .full } [Tutorials](/){ .md-button .md-button--primary .main-button--primary .full }  [Guides](guides){ .md-button .main-button--secondary .full }
```
[the remark this answers is not in the document]
Comment (serinamarie) [reply docs/index.md:42]: @gabcoyne This is still relevant

File: docs/index.md [hunk 5]
```diff
@@ -35,11 +35,11 @@ Trying to implement these features from scratch is a huge pain that takes time,
 </figure>
 
 #### New to Prefect?
-If you're ready to dive in and learn Prefect, check out the Prefect [concepts](/concepts/index/) and try some [tutorials](/tutorials/) to see how they work in action.
+If you're ready to dive in and learn Prefect, check out the Prefect [concepts](/concepts/index/) and try some [tutorials](/) to see how they work in action.
```
[the remark this answers is not in the document]
Comment (serinamarie) [reply docs/index.md:38]: @gabcoyne This is still relevant

File: docs/concepts/projects.md [hunk 6]
```diff
@@ -93,9 +93,9 @@ deployments:
 So long as our `build` steps produce fields called `image_name` and `image_tag`, every time we deploy a new version of our deployment these fields will be dynamically populated with the relevant values.
 
 !!! note "Docker step"
-    The most commonly used build step is [`prefect_docker.projects.steps.build_docker_image`](https://prefecthq.github.io/prefect-docker/projects/steps/#prefect_docker.projects.steps.build_docker_image) which produces both the `image_name` and `image_tag` fields.  
-    
-    For an example, [check out the project tutorial](/tutorials/projects/#dockerized-deployment).
+    The most commonly used build step is [`prefect_docker.projects.steps.build_docker_image`](https://prefecthq.github.io/prefect-docker/projects/steps/#prefect_docker.projects.steps.build_docker_image) which produces both the `image_name` and `image_tag` fields.
+
+    For an example, [check out the project tutorial](/concepts/projects/#dockerized-deployment).
```
[the remark this answers is not in the document]
Comment (serinamarie) [reply docs/concepts/projects.md:98]: @gabcoyne This suggestion still needs committing

File: docs/tutorial/developing-a-new-worker-type.md
[the remark this answers is not in the document]
Comment (serinamarie) [reply docs/tutorial/developing-a-new-worker-type.md:1]: @gabcoyne I think this guy still needs to be removed from `tutorial` since it's in `guides`

**Comment by serinamarie** [comment]
@gabcoyne There are a few suggestions that are still relevant

**Review by serinamarie — APPROVED** [review]

**Review by discdiver — APPROVED** [review]

**Pull request merged**